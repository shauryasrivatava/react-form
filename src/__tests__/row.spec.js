import renderer from 'react-test-renderer';
import Row from '../components/Row';
import React from 'react';
import { mount } from 'enzyme';
import enzymeConfig from '../../enzymeConfig'

describe('<Row />', function(){

    it('should render correctly', function(){
        const component = renderer.create(<Row/>);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('should have <div> with row class', function(){
        const component = mount(<Row />);
        expect(component.find('div').hasClass('row')).toBe(true);
    });
})