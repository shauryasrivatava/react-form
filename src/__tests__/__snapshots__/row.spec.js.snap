// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Row /> should render correctly 1`] = `
<div
  className="row"
/>
`;
